sunpy/tests/setup_command.py

```diff
@@ -83,12 +82,25 @@ def _validate_required_deps(self):
             raise ImportError(
                 "The 'test' command requires the sunpy package to be "
                 "installed and importable.")
+        try:
+            doctestplus = importlib.import_module('pytest_doctestplus')
+            specs = doctestplus.__spec__
```

Comment:
The better way of doing this is with `importlib.util.find_spec` see [this](https://github.com/astropy/astropy/blob/6c8391c8e32138cc102e081c6eaba91a36d5efe5/astropy/conftest.py#L12)

Reply:
actually, I don't think there is a entry point issue here, why can't we just do a 

```
try:
    import pytest_doctestplus
```
?

Reply:
@Cadair I tried the following, but that didn't work for me.
```
        try:
            import pytest_doctestplus
        except ImportError:
            raise ImportError(
                "The 'test' command requires the pytest-doctestplus. Please "
                "install 'pytest-astropy' that provides the required plugin.")
```

Reply:
I used `find_spec` in the first place. Didn't work out at under Py2.7 :disappointed:

Reply:
what didn't work?

Reply:
[this](https://github.com/sunpy/sunpy/pull/2388/commits/754a355833bd663df58271ab4c1650daffe1456d#diff-9019fe4f7f5a889bee5e7b5e091b2e97L88)